code-challenges.test.js

```diff
@@ -39,13 +60,30 @@ var secretCodeWord2 = "gobbledygook"
 var arrayOfWords = ["Apple", "Banana", "Plum", "Cherry", "Kiwi", "Peach"]
 // Expected output: "Apple" "Banana" "Peach"
 
+describe("When wordContainsA is given a array of words,", () => {
 
+    it("returns all the words that contain a letter 'a'.", () => {
+        
+        const actResult = wordContainsA(arrayOfWords)
 
+        expect(actResult).toEqual(["Apple", "Banana", "Peach"])
 
+    })
+})
 
 // b) Create the function that makes the test pass.
 
+const wordContainsA = (array) => {
+
+    return array.filter(word => {
 
+        if(word.includes('a') || word.includes('A')){
+            // if the word contains an a of any form, return the word 
+            return word
+        }
```

Comment:
`.filter()` only needs an arrow function that returns a boolean but because JS also supports "truthy/falsy" behavior, it registers the returned word on line 82 as truthy, so a refactor can look like this: 
```suggestion
    return array.filter(word => word.includes('a') || word.includes('A'));
```